.gitignore

```diff
@@ -0,0 +1 @@
+// Visit https://github.com/github/gitignore to get a gitignore for your project.
```

Comment:
we need to have
- .env
- node_modules

(as a minimum)

Reply:
That is really language specific 😬 . For any .NET language those are not used. Are you ok with choosing one of the gitignore templates after the repo has been created?

Reply:
i generally use global git ignore - but we need to ensure the `.env` is never going to get committed. regarding the other platform related stuff, i concur, node_modules is too specific - let kill that, and picking an ignore template make sense.